dali/operators/decoder/inflate/inflate_params.h

```diff
@@ -0,0 +1,397 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_DECODER_INFLATE_INFLATE_PARAMS_H_
+#define DALI_OPERATORS_DECODER_INFLATE_INFLATE_PARAMS_H_
+
+#include <string>
+#include <vector>
+
+#include "dali/core/backend_tags.h"
+#include "dali/core/common.h"
+#include "dali/core/tensor_shape.h"
+#include "dali/pipeline/data/types.h"
+#include "dali/pipeline/operator/arg_helper.h"
+#include "dali/pipeline/operator/operator.h"
+
+namespace dali {
+
+namespace inflate {
+
+constexpr static const char *dTypeArgName = "dtype";
+constexpr static const char *algArgName = "algorithm";
+constexpr static const char *shapeArgName = "shape";
+constexpr static const char *offsetArgName = "chunks_offsets";
+constexpr static const char *sizeArgName = "chunks_sizes";
+constexpr static const char *layoutArgName = "layout";
+constexpr static const char *sequenceLayoutArgName = "sequence_extent";
```

Comment:
As mentioned before, I consider this actually harmful. Code like this is harder to read and harder to debug. Also, there's a tendency for these names to drift away. Actually, it seems that this already happened here with
`sequenceLayoutArgName = "sequence_extent";` - is it `extent` or `layout`?

Reply:
After reading the PR, this argument should be called `sequence_axis` (or perhaps even `sequence_axis_name`) - that's the nomenclature we use in other operators that deal with axis labels.

Reply:
Thanks, I renamed the arg.